examples/index.html

```diff
@@ -489,13 +512,42 @@ <h3>Translating texts</h3>
       id = 'autocomplete-defaultValue'
       accessibleAutocomplete({
         defaultValue: 'Germany',
+        element: element,
+        id: id,
+        source: countries,
+        menuAttributes: {
+          "aria-labelledby": id
+        }
+      })
+    </script>
+
+    <script type="text/javascript">
+      element = document.querySelector('#tt-inputClasses')
+      id = 'autocomplete-inputClasses'
+      accessibleAutocomplete({
         element: element,
         id: id,
         source: countries,
         menuAttributes: {
           "aria-labelledby": id
         },
-        inputClasses: "govuk-input"
+        autoselect: true,
+        inputClasses: 'custom-input-class'
```

Comment:
Worth using an [`app-*` prefix](https://github.com/alphagov/govuk-frontend/blob/b44022f0ddb62268ee608c699670bc93529140fc/docs/contributing/coding-standards/css.md#govuk-namespacing) for a (future) `govuk-input app-input` suggestion?

```suggestion
        inputClasses: 'app-input'
```